packages/embed/src/factories/create-widget/elements/build-widget.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { buildWidget } from './build-widget'
+
+describe('build-widget', () => {
+  describe('#buildWidget', () => {
```

Comment:
What's up with this context nesting? Is It filename->function?

Reply:
Correct. This file has just one function, but it could have more if necessary.